src/components/pages/Stake/createLock/CreateLockConfirm.tsx

```diff
@@ -108,21 +109,21 @@ export const CreateLockConfirm: FC = () => {
           >
             You would start with{' '}
             {simUserLockup?.bias.simple &&
-            simUserLockup.bias.simple > 0 &&
-            totalSupply &&
-            totalSupply.simple > 0 ? (
-              <SimulatedCountUp
-                end={
-                  (simUserLockup.bias.simple /
-                    (totalSupply.simple + simUserLockup.bias.simple)) *
-                  100
-                }
-                decimals={6}
-                suffix=" %"
-              />
-            ) : (
-              '-'
-            )}{' '}
+              simUserLockup.bias.simple > 0 &&
+              totalSupply &&
+              totalSupply.simple > 0 ? (
+                <SimulatedCountUp
+                  end={
```

Comment:
What if the value here isn't greater than the existing one? Probably best to use a red colour instead of green, so the user is informed.

There is a `highlightColor` prop for the `CountUp` component that can be used instead of the `SimulatedCountUp` styled component.